### PR TITLE
[FEA] First simple implementation of Sensor

### PR DESCRIPTION
**Pull request by lucasclopesr**
Simple implementation of random generating sensor that publishes its readings to a channel.

### CODE REVIEW

File: src/sensor.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	ch := make(chan string)
+
+	go getReadings(ch)
+	fmt.Println(<-ch)
+}
+
+func getReadings(ch chan<- string) {
+	// generate readings
+	start := time.Now()
+  rand.Seed(start.UTC().UnixNano())
```
Comment (GabrielNegreirosLima) [inline src/sensor.go:19]: We have to align our tab indentation :s

Comment (lucasclopesr) [reply src/sensor.go:19]: I propose we use 2 tab spacing since it makes it harder to reach a width threshold

Comment (GabrielNegreirosLima) [reply src/sensor.go:19]: That sounds nice. Can you change it for us? I'll set it in my vscode.

Comment (lucasclopesr) [reply src/sensor.go:19]: Alright!

**Pull request merged**